resources/js/Components/Main/TitleLogo.tsx

```diff
@@ -0,0 +1,40 @@
+import { useLaravelReactI18n } from 'laravel-react-i18n';
+import ApplicationLogo from '@/Components/ApplicationLogo';
+import Button from '@/Components/Shared/Button';
+import { Link } from '@heroui/link';
+
+export function TitleLogo() {
+    const { t } = useLaravelReactI18n();
+
+    return (
+        <div className="flex flex-col w-[60rem] h-[26rem] justify-self-start gap-41 px-14 py-12
+            bg-[url(/images/title-logo.png)] bg-cover bg-center opacity-85 mt-20 ml-56 ">
```

Comment:
- Remake background with [Image](https://www.heroui.com/docs/components/image) for loading animations
- There's no need for margins
- Why it contains opacity?
```suggestion
            ">
```

Reply:
ok

Reply:
With transparency, the element is more consistent with Visualization in Figma and also transparency as a parameter is in Figma